.python.env

```diff
@@ -0,0 +1 @@
+PYTHONPATH=${PYTHONPATH}:./extlibs:/Applications/QGIS-LTR.app/Contents/MacOS/lib/python3.9
```

Comment:
I don't love hardcoding these paths into our repo but I think we can refine that later.  These are simply for developer experience so will just be an annoyance for any plugin developers with differing paths.